tests/test_basic.py

```diff
@@ -192,6 +192,42 @@ def test_default_list(tmpdir, runner, create):
     assert todo.summary == 'test default list'
 
 
+def test_default_due_is_not_specified(tmpdir, runner, create):
```

Comment:
You can collapse all these tests into one using http://doc.pytest.org/en/latest/example/parametrize.html